tgui/packages/tgui/interfaces/Uplink.js

```diff
@@ -32,15 +33,21 @@ export const Uplink = (props, context) => {
           <Tabs.Tab
             key="PurchasePage"
             selected={tabIndex === 0}
-            onClick={() => setTabIndex(0)}
+            onClick={() => {
+                setTabIndex(0);
+                setSearchText("");
```

Comment:
why bother setting search text to a blank string when its only used for the equipment page?

Reply:
The search is also used for the exploitable information page